### DISCUSSION TITLE
XPath - Return results as a list

### DISCUSSION THREAD

**Discussion by Cloud-Weaver** [swhitley/ERPHelper#26] [Ideas]
It would be nice for the XPath feature to return a list of matching results in the plug-in window. This would be helpful for a user to execute a path and be able to review the results and verify the XPath is working as expected. 

Currently the XPath feature highlights in gray but if the user scrolls down the highlighting is lost. The utility of XPath is currently limited to confirming that an XPath exists but not if multiple matches occur and where across the whole file.
![image](https://user-images.githubusercontent.com/69214198/128380989-fa51b058-e3f3-4a80-8fbf-ba4f8e83b9b3.png)

It would be helpful to use something like the "Find" feature that highlights the results in a distinct color. 

![image](https://user-images.githubusercontent.com/69214198/128380458-4ddfeddb-8b45-40a9-a858-3d98723450e1.png)

Alternatively the plug-in could list out results in the way that Excel does when a user performs "Find All" (it returns all matches, a preview of cell value, and the location of each result).

**Comment by swhitley** [comment]
Addressed partially in v1.1.2.  Annotations show inline results.  Future decision to enable result list is pending.
